pyproject.toml

```diff
@@ -25,7 +25,7 @@ version = "0.5.1"
 bite-parser = "^0.1.1"
 dataclasses-serialization = "^1.3.1"
 prometheus_client = "^0.11.0"
-python = "^3.7,<3.10"
+python = "^3.7,<3.11"
```

Comment:
I think I would prefer removing the upper version bound completely. [It was initially introduced because a dependency was incompatible with Python 3.9 without properly declaring this itself.](https://github.com/jgosmann/dmarc-metrics-exporter/commit/b4134d89b32019fdb86a949fa4bae8fadb047888) This dependency is no longer in use and I think I just forgot to remove the upper bound when dropping the dependency. At least I see no other reason to keep it given that minor Python releases usually do not break existing code.

Reply:
Makes sense. I'll close this PR then.